neurokit2/complexity/fractal_dimension.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+
+
+def fractal_dimension(signal, method="higuchi", kmax=5):
+    """
+    Computes fractal dimension according to Higuchi's Fractal Dimension (HFD) or
+    Katz's Fractal Dimension (KFD).
+    
+    Parameters
+    ----------
+    signal : Union[list, np.array, pd.Series]
+        The signal (i.e., a time series) in the form of a vector of values.
+    method : str
+        The algorithm used to compute frctal dimension, HFD (by calling "hfd" or "higuchi") or
+        KFD (by calling "kfd" or "katz").
+    
+    Returns
+    ----------
+    float
+        The fractal dimension.
+
+    Examples
+    ----------
+    >>> import neurokit2 as nk
+    >>>
+    >>> signal = nk.signal_simulate(duration=2, frequency=5, noise=10)
+    >>>
+    >>> hfd = nk.fractal_dimension(signal, method="higuchi")
+    >>> hfd #doctest: +SKIP
+    >>> kfd = nk.fractal_dimension(signal, method="katz")
+    >>> kfd #doctest: +SKIP
+
+    References
+    ----------
+    - Higuchi, T. (1988). Approach to an irregular time series on the basis of the fractal theory.
+    Physica D: Nonlinear Phenomena, 31(2), 277-283.
+
+    - Katz, M. J. (1988). Fractals and the analysis of waveforms.
+    Computers in Biology and Medicine, 18(3), 145–156. doi:10.1016/0010-4825(88)90041-8.
+    """
+
+    if method in ["higuchi", "hfd"]:
+        dimension = _fractal_dimension_higuchi(signal, kmax)
+    elif method in ["katz", "katz's", "kfd"]:
+        dimension = _fractal_dimension_katz(signal)
+
+    return dimension
+        
+
+def _fractal_dimension_higuchi(signal, kmax=5):
+    """Computes Higuchi's Fractal Dimension (HFD) from the time series.
+    Value should fall between 1 and 2.
+    
+    For more information about k parameter selection, see https://ieeexplore.ieee.org/document/7168285
```

Comment:
```suggestion
    For more information about k parameter selection, see the papers referenced below.
    
    References
    ------------
    - ref + link
```

Reply:
That said, we should probably find a way of having some `complexity_k()` optimizer function